docs/user-guide/tips-tricks-and-anti-patterns.ipynb

```diff
@@ -226,7 +226,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In the case of only two neighbors, the same could be achieved using slicing with strides, however scipp does not support strides yet.\n",
+    "In the case of only two neighbors, the same could be achieved using slicing with a stride.\n",
```

Comment:
Do you want to provide an example?

Reply:
I don't think so, since that is not the point here? It would distract from what we are trying to say?

Reply:
Then we should remove the comment altogether.